docs/sdkx-react-native/notifications-ios.mdx

```diff
@@ -0,0 +1,285 @@
+
+<Admonition type="info" title="Note">
+
+Your Push Certificate has an expiry date, as indicated in the below screenshot. Helpshift will not send you a reminder when your Push Certificate expires, so please make sure that your developer keeps a tab on the expiry date to re-upload the Push Certificate.
+
+</Admonition>
+
+<Image
+  src="/static/books/sdkx_ios/cert-expiry.png"
+  width="full"
+  alt="cert-expiry.png"
+/>{" "}
+
+### Configure the Helpshift React Native Plugin SDK to handle notifications {#configure-helpshift-react-native-sdk}
+
+After you have configured the push notifications in the Helpshift console, you will need to add additional setup in the project, too. First of all, you should enable the Use Push Notifications toggle in the Project Settings.
+
+Here we are providing a sample example of push notification use, [react-native-notifications](https://www.npmjs.com/package/react-native-notifications) package. You can use any package of your choice.
+
+To setup notifications using react-native-notifications, please follow [this](https://wix.github.io/react-native-notifications/docs/getting-started)
+
+Example :
+
+**App.tsx**
+
+```javascript
+import {
+  handleProactiveLink,
+  handlePush,
+  registerPushToken,
+} from 'helpshift-plugin-sdkx-react-native';
+import {
+  Notification,
+  Notifications,
+  Registered,
+  RegistrationError,
+} from 'react-native-notifications';
+```
+
+#### For Register notifications
+
+```javascript
+  Notifications.registerRemoteNotifications();
+
+  //Get Callback for registerRemoteNotifications.
+  Notifications.events().registerRemoteNotificationsRegistered(
+    (event: Registered) => {
+      registerPushToken(event.deviceToken);
+    }
+  );
+```
+
+#### For handle notification received in foreground
+
+```javascript
+Notifications.events().registerNotificationReceivedForeground(
+    (notification: Notification, completion) => {
+      const notificationData = notification.payload;
+
+      const doesNotificationDataHaveProactiveLink =
+        notificationData.hasOwnProperty('helpshift_proactive_link');
+      const doesNotificationDataHaveOrigin =
+        notificationData.hasOwnProperty('origin') &&
+        notificationData?.origin == 'helpshift';
+
+      if (doesNotificationDataHaveProactiveLink) {
+        completion({ alert: true, sound: false, badge: false });
+      } else if (doesNotificationDataHaveOrigin) {
+        handlePush(notificationData, false);
+        completion({ alert: false, sound: false, badge: false });
+      } else {
+        completion({ alert: true, sound: true, badge: false });
+      }
+    }
+  );
+```
+
+####  When user tap on notifications
+
+```javascript
+Notifications.events().registerNotificationOpened(
+    (notification: Notification, completion: () => void) => {
+
+      const notificationData = notification.payload;
+      //This is for manage outbond support link.
+      const doesNotificationDataHaveProactiveLink =
```

Comment:
We provided this is how we handle the proactive key inside a notification, in case, any developer use `react-native-notifications` this is complete code.